### PR TITLE
[msg] event listeners/handlers added

### PR DESCRIPTION
**Pull request by alicemeeyoung**

### CODE REVIEW

File: rest-server/src/components/auth/authQueries.js [hunk 1]
```diff
@@ -26,7 +27,8 @@ export const loginQuery = async (body) => {
     const queryString = loginHelper(body);
     const data = await db.queryAsync(queryString);
     // db.end();
-    success('loginQuery - successfully retrieved data ', data);
+    console.log('data', data.rows[0].username)
```
Comment (jameshyen) [inline rest-server/src/components/auth/authQueries.js]: Plz delete the console logs. :)

File: socket-server/src/index.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ const server = http.createServer();
 const io = SocketIo(server);
 const rooms = new Rooms(io);
 
+
```
Comment (jameshyen) [inline socket-server/src/index.js]: Y the extra line.

**Comment by jameshyen** [comment]
Please rebase, also try not to (if you would) change the way they originally did the `success` messages (although it is true that a lot of them output `[object Object]`. Also please delete the console logs and eliminate the extra lines.

File: rest-server/src/components/auth/authQueries.js (hunk 1 above)
Comment (alicemeeyoung) [reply rest-server/src/components/auth/authQueries.js]: b

**Pull request closed without merging**